lib/kino/js/live_server.ex

```diff
@@ -16,10 +16,8 @@ defmodule Kino.JS.LiveServer do
   defdelegate call(pid, term, timeout), to: GenServer
 
   def broadcast_event(ctx, event, payload) do
-    for pid <- ctx.__private__.client_pids do
-      send(pid, {:event, event, payload, %{ref: ctx.__private__.ref}})
-    end
-
+    ref = ctx.__private__.ref
+    Kino.Bridge.broadcast("js_live:#{ref}", {:event, event, payload, %{ref: ref}})
```

Comment:
I went with a more specific topic and allowed subscribing to `js_live:*` on Livebook side. In the channel we want all of those for the given session, but I can imagine subscribing to specific output only.

Reply:
My only concern is that this obfuscates what topic we subscribe to on Livebook side, but that's internal so it's probably fine. Does it still makes sense if we subscribe to `js_live:<ref>` in the channel though?

Reply:
I would separate just to give us more control. Think about it like this: the fact it uses a pubsub on the livebook server should be an implementation detail from Kino perspective. So the arguments should not necessarily be mirror after pubsub.

Reply:
Sounds good!